### PR TITLE
Improve the shading language reference table

### LINKED ISSUE #4274: Linear and Hermite interpolate don't specify which value is used for interpolation
**Your Godot version:**
3.2.3

**Issue description:**
Linear and Hermite interpolate don't specify which value is used for interpolation.
This is confusing for beginners like me.

**Current**
Linear interpolate
Hermite interpolate

**Suggested**
Linear interpolate between a and b by c
Hermite interpolate between a and b by c

**URL to the documentation page (if already existing):**
https://docs.godotengine.org/en/stable/tutorials/shading/shading_reference/shading_language.html#built-in-functions

### PR DESCRIPTION
**Pull request by Calinou**
This closes #4274.

### CODE REVIEW

File: tutorials/shaders/shader_reference/shading_language.rst [hunk 1]
```diff
@@ -722,55 +722,61 @@ When vec_type (float), vec_int_type, vec_uint_type, vec_bool_type nomenclature i
...
+| ivec_type **sign** (ivec_type x)                                       | Sign (returns ``1`` if positive, ``-1`` if negative,          |
+|                                                                        | ``0`` if zero)                                                |
 +------------------------------------------------------------------------+---------------------------------------------------------------+
-| vec_type **floor** (vec_type x)                                        | Floor                                                         |
+| vec_type **floor** (vec_type x)                                        | Round to the integer below                                    |
 +------------------------------------------------------------------------+---------------------------------------------------------------+
-| vec_type **round** (vec_type x)                                        | Round                                                         |
+| vec_type **round** (vec_type x)                                        | Round away from zero                                          |
```
Comment (Calinou) [inline tutorials/shaders/shader_reference/shading_language.rst:737]: I don't know if this is vendor-specific. Maybe "Round to the nearest integer" is a safer choice here. @cc clayjohn

File: tutorials/shaders/shader_reference/shading_language.rst [hunk 2]
```diff
@@ -822,9 +828,9 @@ When vec_type (float), vec_int_type, vec_uint_type, vec_bool_type nomenclature i
 +------------------------------------------------------------------------+---------------------------------------------------------------+
 | bvec_type **notEqual** (vec_type x, vec_type y)                        | Bool vector comparison on != int/uint/float vectors           |
 +------------------------------------------------------------------------+---------------------------------------------------------------+
-| bool **any** (bvec_type x)                                             | Any component is ``true``                                     |
+| bool **any** (bvec_type x)                                             | ``true`` if component is ``true``, ``false`` otherwise        |
 +------------------------------------------------------------------------+---------------------------------------------------------------+
-| bool **all** (bvec_type x)                                             | All components are ``true``                                   |
+| bool **all** (bvec_type x)                                             | ``true`` components are ``true``, ``false`` otherwise         |
```
Comment (clayjohn) [inline tutorials/shaders/shader_reference/shading_language.rst:833]: "true" if all components are...

File: tutorials/shaders/shader_reference/shading_language.rst [hunk 3]
```diff
@@ -822,9 +828,9 @@ When vec_type (float), vec_int_type, vec_uint_type, vec_bool_type nomenclature i
 +------------------------------------------------------------------------+---------------------------------------------------------------+
 | bvec_type **notEqual** (vec_type x, vec_type y)                        | Bool vector comparison on != int/uint/float vectors           |
 +------------------------------------------------------------------------+---------------------------------------------------------------+
-| bool **any** (bvec_type x)                                             | Any component is ``true``                                     |
+| bool **any** (bvec_type x)                                             | ``true`` if component is ``true``, ``false`` otherwise        |
```
Comment (clayjohn) [inline tutorials/shaders/shader_reference/shading_language.rst:831]: True if any component...

**Review by clayjohn — CHANGES_REQUESTED** [review]

**Comment by NathanLovato** [comment]
Thanks!

**Pull request merged**